lib/search_vector.rb

```diff
@@ -5,10 +5,15 @@ module SearchVector
   AcceptedTables = ['supporters', 'payments']
 
   def self.query(query_string, expr=nil)
-    (expr || Qexpr.new).where(
+    query = if query_string.is_a?(Integer) || query_string.is_int?
+      "(supporters.fts @@ websearch_to_tsquery('english', $search::varchar(255))
+      OR donations.fts  @@ websearch_to_tsquery('english', $search::varchar(255))
```

Comment:
Is there any reason we couldn't move these two out lines out since they're the same between each branch of the boolean and then only append the `OR payments.id = $search::INTEGER` to the statement if the boolean is true?

Reply:
I thought it looked better this way, but I don't know. I did it as you suggested now, what do you think?

Reply:
Haha, I looked at it now and I agree with you, it looked better before. Sorry!

Reply:
Hahaha I'll change it back